src/EventManager.php

```diff
@@ -54,20 +64,25 @@ class EventManager implements EventManagerInterface
      *
      * @param  null|string|int|array|Traversable $identifiers
```

Comment:
It's not supposed to; I've changed it already on my local copy to array only. (Since it delegates to `setIdentifiers()`, which also typehints on array at this time. You'll see that change shortly.